### ISSUE TITLE
Personal Data Retention for Refunded Orders

### ISSUE THREAD

**Issue by SiobhyB** [woocommerce/woocommerce#28800]
**Describe the solution you'd like**

If you navigate to _WooCommerce_ > _Settings_ > _Accounts & Privacy_ > _Personal data retention_, there is currently no option for how long to retain details associated with refunded orders. This was brought up by a customer in [this forum post](https://wordpress.org/support/topic/remove-personal-data-gdpr-for-refunded-orders/) who would like a way to automate the deletion of data on file for refunded orders.

**Describe alternatives you've considered**

I'm not sure if this would be a required setting for GDPR or not, so creating this issue for further discussion amongst those more knowledgable on privacy matters than myself.

**Additional context**

You can see a screenshot of the current settings under _Personal data retention_ here:

<img width="994" alt="Screenshot 2021-01-12 at 15 45 00" src="https://user-images.githubusercontent.com/2998162/104337234-28988a80-54ed-11eb-929f-8d778e55d28c.png">

**Comment by masteradhoc** [comment]
@SiobhyB Currently i see the following statues are missing: Refunded, On-Hold, Processing
While i think it makes sense to not offer this for processing i think for on-hold and also refunded woocommerce should offer this also.

**Comment by Konamiman** [comment]
Hi @masteradhoc, we came across this as part of our repository maintenance tasks. I see that you created https://github.com/woocommerce/woocommerce/pull/28803 but then you closed it, could you please tell us why?

**Comment by masteradhoc** [comment]
@Konamiman thanks for the comment. was such a long time ago sorry i dont remember :) should i open it again for you guys to check?

**Comment by Konamiman** [comment]
@masteradhoc Yes please. Either reopen, or if after that much time there are too many merge conflicts, create a new one referencing this issue. Thank you.

**Comment by masteradhoc** [comment]
@Konamiman could you give #55514 a check? Thanks!
